### ISSUE TITLE
Blued international 

### ISSUE THREAD

**Issue by mc90blue** [lack21115/BluedHook#7]
Do you have any intention to develop something similar to the international version (com.blied.international)? Thanks.

**Comment by lack21115** [comment]
OK, I will try.

**Comment by mc90blue** [comment]
Yes, I'd love to. Sorry for the late reply. Can you DM me the module?

**Comment by lack21115** [comment]
Latest
